package.json

```diff
@@ -9,11 +9,13 @@
     "test": "jest --watch"
   },
   "dependencies": {
+    "react-dom": "16.13.1",
     "next": "9.4.2",
     "react": "16.13.1",
-    "react-dom": "16.13.1"
+    "data-explorer": "git+https://github.com/datopian/data-explorer.git"
```

Comment:
@lauragift21 I guess you're updating `package.json` file manually - if so I wouldn't recommend doing so. You can run `yarn add package` which will automatically update list if dependencies.